students/Lincoln_Zhang/Session2/grid_print.py

```diff
@@ -0,0 +1,53 @@
+#
+#Session 2 Class excercise
+#print grid like this
+#+ - - - - + - - - - +
+#|         |         |
+#|         |         |
+#|         |         |
+#|         |         |
+#+ - - - - + - - - - +
+#|         |         |
+#|         |         |
+#|         |         |
+#|         |         |
+#+ - - - - + - - - - +
+
+def print_plus():
+	print ("+",end="")
+
+def print_minus():
+	print(" - - - - ",end="")
```

Comment:
The parameter we are passing into grid_print should determine the size. It should not be a hardcoded amount of 4 minuses. It looks like you are calculating the "size" of the grid to be the length from top to down. Consider it to be the full length top down and across so a size of 3 should be 3x3.